source/index.md

```diff
@@ -340,27 +342,30 @@ them to provide card information for subsequent payments.
 
 ### Subscription concept
 
-Many payment gateways offer a subscription concept where a card can be
-subscribed for recurring payments. This is supported in our API using card
-tokens. Card tokens can be created [explicitly](#tokenize-a-card) or implicitly
-when the first [authorization is created](#reserve-money).
-
-Actual recurring transactions must be made by
-[creating an authorization](#authorizations).
-
+Many PSPs have a subscription concept for supporting recurring payments.
+The first approved authorization is the initial recurring authorization (also
+known as "first in series"), all later authorizations are called subsequent
+authorizations. Clearhaus supports subscriptions in the form of recurring
+transactions.
 
 ### Repeatedly reserve money
 
-A recurring payment is made by making an authorization based on a card
-resource and setting `recurring` parameter to `true`:
+A recurring payment is made by making an authorization and setting `recurring`
```

Comment:
Perhaps we confuse the reader with too many terms for the same thing? In line 349 we have "recurring transactions"; here we have "recurring payment".

Reply:
@kse-clearhaus  WDYT about `s/payment/transaction/`?